.github/workflows/run-test-on-pr.yml

```diff
@@ -10,31 +10,43 @@ jobs:
     environment: test # The below environment variables are only available in the test environment
     
     env:
-      TEST_API_KEY: ${{ secrets.TEST_API_KEY }}
```

Comment:
api key is dynamically generate now.

Reply:
The TEST_API_KEY on staging is the key from the staging EigenLayer-AVS, so let’s keep it.

Reply:
updated